src/pages/settings/index.tsx

```diff
@@ -0,0 +1,96 @@
+import type { NextPage } from "next";
+
+import Image from "next/image";
+import Link from "next/link";
+import { Text, Group } from "@mantine/core";
+
+const SETTINGS = [
+  {
+    label: "プロフィール",
+    // path: "/profile",
+    path: "#",
+    icon: "/arrow.svg",
+  },
+  {
+    label: "アカウント",
+    // path: "/account",
+    path: "#",
+    icon: "/arrow.svg",
+  },
+  {
+    label: "テーマ",
+    // path: "/theme",
+    path: "#",
+    icon: "/arrow.svg",
+  },
+];
+
+const SUPPORTS = [
+  {
+    label: "プライバシーポリシー",
+    // path: "/policy",
+    path: "#",
+    icon: "/arrow.svg",
+  },
+  {
+    label: "利用規約",
+    // path: "/rules",
+    path: "#",
+    icon: "/arrow.svg",
+  },
+  {
+    label: "お問い合わせ",
+    // path: "/question",
+    path: "#",
+    icon: "/externalLink.svg",
+  },
+];
```

Comment:
```suggestion
const SUPPORTS = [
  {
    label: "プライバシーポリシー",
    // path: "/policy",
    path: "#",
    icon: "/arrow.svg",
  },
  {
    label: "利用規約",
    // path: "/rules",
    path: "#",
    icon: "/arrow.svg",
  },
  {
    label: "お問い合わせ",
    // path: "/question",
    path: "#",
    icon: "/externalLink.svg",
  },
] as const;
```
`SUPPORTS` は定数を意図しているかと思いますが、
書き換え可能になってるので const assertion しておくとよさそうですね。

Reply:
# FYI
## assertion なし
<img width="610" alt="image" src="https://user-images.githubusercontent.com/17216462/163658798-cf87a926-4e79-43cb-a0a9-10b354a41119.png">

Reply:
## assetion あり
<img width="744" alt="image" src="https://user-images.githubusercontent.com/17216462/163658855-e5d01bd2-3f80-44bb-954b-e3eb281ecf94.png">